scripts/zones/Cape_Teriggan/Zone.lua

```diff
@@ -11,10 +11,9 @@ require('scripts/globals/zone')
 local zoneObject = {}
 
 zoneObject.onInitialize = function(zone)
+    -- NM Persistence
     local kreutzet = GetMobByID(ID.mob.KREUTZET)
-
-    UpdateNMSpawnPoint(ID.mob.KREUTZET)
-    kreutzet:setRespawnTime(math.random(32400, 43200)) -- 9 to 12 hours
+    xi.mob.nmTODPersistCache(zone, ID.mob.KREUTZET)
     kreutzet:setLocalVar("cooldown", os.time() + kreutzet:getRespawnTime() / 1000)
     DisallowRespawn(kreutzet:getID(), true) -- prevents accidental 'pop' during no wind weather and immediate despawn
 
```

Comment:
Tell me if I am going crazy but why is this dividing the respawn time by 1000? We should just use the serverVar being set in the new nmpersist and call that in the zone lua as well. Same as KV

Reply:
Should be fixed